app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -292,20 +292,7 @@ void main() {
       flutterRepoPRs.add(prInProgress);
       await tester.get(handler);
       _verifyQueries();
-      githubGraphQLClient.verifyMutations(<MutationOptions>[
```

Comment:
These tests are actually expecting the wrong thing. They didn't set a status, and are reproducing the bug Cocoon has now. Instead, we expect no more graphql mutations as it shouldn't leave a comment.